docs/integrations/index.md

```diff
@@ -18,44 +18,79 @@ To determine if an app is supported by your pricing plan, see [Apps and Sumo Log
 <div className="box-wrapper" markdown="1">
 <div className="box smallbox1 card">
   <div className="container">
-  <img src='https://upload.wikimedia.org/wikipedia/commons/9/93/Amazon_Web_Services_Logo.svg' alt="Thumbnail icon" width="65"/>
-  <h3><a href="/docs/integrations/amazon-aws/">Amazon / AWS</a></h3>
-  <p>A guide to our apps for Amazon and AWS products.</p>
+  <img src={useBaseUrl('img/integrations/web-servers/apache.png')} alt="Thumbnail icon" width="50"/>
+  <h3><a href="/docs/integrations/web-servers/apache">Apache</a></h3>
+  <p>A guide to the Sumo Logic app for Apache.</p>
```

Comment:
This is the main integrations doc. Did you mean to put all of these in the web servers doc?

Reply:
Yes, I was working on the web servers index. Needed to pull to the web servers doc.